src/interpreter.rs

```diff
@@ -166,6 +166,38 @@ impl Interpreter {
         let model = Model::new(model_path)?;
         Interpreter::new(&model, options)
     }
+    /// Creates an [`Interpreter`] with model bytes.
+    ///
+    /// # Arguments
+    ///
+    /// * `model_bytes`: Path to TensorFlow Lite model
+    /// * `options`: Interpreter [`Options`]
+    ///
+    /// returns: Result<Interpreter, Error>
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use tflitec::interpreter::Interpreter;
+    /// use std::path::MAIN_SEPARATOR;
+    ///
+    /// let path = format!("tests{}add.bin", MAIN_SEPARATOR);
+    /// let interpreter = Interpreter::with_model_path(&path, None)?;
```

Comment:
Unnacceptable laziness. However, as I'm not certain if this change is desired, I'll wait till I know to fix it. It's one line. Might also need tests, but I defer to the repo owners